### PR TITLE
update newtonsoft.json dependencies

### PR DESCRIPTION
**Pull request by lyndaidaii**
WindowsAzure.Storage
9.3.3  has dependency of newtonsoft. Update newtonsoft.json to 13.0.1 avoid bring low version newtonsoft

Address: https://github.com/NuGet/Engineering/issues/4452

### CODE REVIEW

**Review by agr — APPROVED** [review]

File: src/NuGet.Services.Configuration/NuGet.Services.Configuration.csproj [hunk 1]
```diff
@@ -21,6 +21,9 @@
     <PackageReference Include="Microsoft.Extensions.Options">
       <Version>2.2.0</Version>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>13.0.1</Version>
```
Comment (joelverhagen) [inline src/NuGet.Services.Configuration/NuGet.Services.Configuration.csproj:25]: This should not be necessary with the transitive dependency in KeyVault.csproj, right?

File: tests/NuGet.Services.Storage.Tests/NuGet.Services.Storage.Tests.csproj [hunk 2]
```diff
@@ -8,6 +8,9 @@
     <PackageReference Include="Moq">
       <Version>4.10.0</Version>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>13.0.1</Version>
```
Comment (joelverhagen) [inline tests/NuGet.Services.Storage.Tests/NuGet.Services.Storage.Tests.csproj:12]: This should not be necessary due to the transitive dependency in Storage.csproj right?

**Review by joelverhagen — CHANGES_REQUESTED** [review]
⌚ we can prune the changes a bit I think.

**Comment by lyndaidaii** [comment]
@joelverhagen, please take another look. Thanks.

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**